src/components/core/Slider/Slider.tsx

```diff
@@ -48,7 +48,7 @@ const Slider = ({ label, markers, maxW, ...rest }: Props) => {
           <PrimitiveSliderRange className={classNames.range} />
         </PrimitiveSliderTrack>
 
-        <PrimitiveSliderThumb className={classNames.thumb} />
+        <PrimitiveSliderThumb index={0} className={classNames.thumb} />
```

Comment:
Interesting, it seems that the [ark documentation](https://ark-ui.com/docs/components/slider) also passes a `key` to the `SliderThumb`. We may want to look deeper into the reasoning behind this / determine if that is need or good practice to have.

Reply:
`index` is useful in case you have multiple slider thumbs (note that `RangeSlider` was deprecated in favor of this single Ark `Slider` component with v1). I believe `key` is not required, we will want to avoid it unless we need it anyways since it is a[ React mechanism](https://legacy.reactjs.org/warnings/special-props.html) that can cause unnecessary rerenders.